aiohttp_jwt/__init__.py

```diff
@@ -1,5 +1,5 @@
 
-__version__ = '0.3.0'
+__version__ = '0.3.1'
```

Comment:
it should be feature release i guess so `0.4.0`

Reply:
I will remove it from my pull request.